test/e2e/helpers.go

```diff
@@ -73,31 +73,21 @@ kind: ImagesLock
 const imageFile = "images.yml"
 const bundleFile = "bundle.yml"
 
-func createBundleDir(dir, bYml, iYml string) (string, error) {
-	imgpkgDir := filepath.Join(dir, ".imgpkg")
-	err := os.Mkdir(imgpkgDir, 0700)
-	if err != nil {
-		return "", err
-	}
-
-	fileContents := map[string]string{
-		bundleFile: bYml,
-		imageFile:  iYml,
-	}
-	for filename, contents := range fileContents {
-		err = ioutil.WriteFile(filepath.Join(imgpkgDir, filename), []byte(contents), 0600)
-		if err != nil {
-			return imgpkgDir, err
-		}
-	}
-	return imgpkgDir, nil
-}
-
-func extractDigest(out string, t *testing.T) string {
+func extractDigest(t *testing.T, out string) string {
 	t.Helper()
 	match := regexp.MustCompile("@(sha256:[0123456789abcdef]{64})").FindStringSubmatch(out)
 	if len(match) != 2 {
 		t.Fatalf("Expected to find digest in output '%s'", out)
 	}
 	return match[1]
 }
+
+var letterRunes = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789")
+
+func randStringRunes(n int) string {
```

Comment:
randStringRunes -> randString? (not sure what rune is describing to the consumer of this function).